src/main/java/com/regnosys/rosetta/common/transform/TestPackUtils.java

```diff
@@ -40,11 +40,6 @@
 
 public class TestPackUtils {
     public static final Path PROJECTION_PATH = Paths.get(TransformType.PROJECTION.getResourcePath());
-    public static final Path PROJECTION_ISO20022_PATH = PROJECTION_PATH.resolve("iso-20022");
-
-    @Deprecated //This should not be used as the new test pack configs are not generated in ISO-20022 folder
-    public static final Path PROJECTION_CONFIG_PATH = PROJECTION_ISO20022_PATH.resolve("config");
-    public static final Path PROJECTION_OUTPUT_PATH = PROJECTION_ISO20022_PATH.resolve("output");
 
```

Comment:
Is it not possible to get rid of these nasty methods?

```
    public static String getProjectionTestPackName(String reportId) {
        return "test-pack-projection-" + reportId + "-report-to-iso20022.*\\.json";
    }

    public static String getReportTestPackName(String reportId) {
        return "test-pack-report-" + reportId + ".*\\.json";
    }
```

Reply:
Apparently used by DRR validation summary, @PayalKhanna will fix in a separate PR